x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_form.tsx

```diff
@@ -168,7 +169,7 @@ export const AlertForm = ({
     : null;
 
   const alertTypeRegistryList =
-    alert.consumer === 'alerts'
+    alert.consumer === AlertsFeatureId
```

Comment:
As the consumer is now linked to privileges - it's important that these are aligned and so instead of a string this is now a shared common constant.